subcontracted_service/README.rst

```diff
@@ -0,0 +1,83 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+=======================================
+Trigger procurement for service product
+=======================================
+
+This module adds the functionality of being able to automatically make a
+purchase order (through procurements) when a service product if sold
```

Comment:
This module can be independent of sales. We suggest that you change the description to just say that this module supports the creation of procurements that contains service products. Then, you can also say that this could be useful in the sale of services that are to be subcontracted.